ci-operator/step-registry/rhcs/e2e/setup/rhcs-e2e-setup-commands.sh

```diff
@@ -34,6 +34,11 @@ if [ -z "${RHCS_TOKEN}" ]; then
 fi
 export RHCS_TOKEN=${RHCS_TOKEN}
 export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred
+
+if [[ ${ENABLE_SHARED_VPC} == "yes" ]]; then
```

Comment:
why not using `true|false` ?

Reply:
`ENABLE_SHARED_VPC` is being used by OCP as well, to keep the parameter consistent in this repo, `yes|no` is used.